source/utils/events.js

```diff
@@ -1,36 +1,36 @@
 export default {
-  getMousePosition (event) {
+  getMousePosition(event) {
```

Comment:
In Daedalus we enforce a space between function name and the parans … would be good to keep it consistent between projects!